### PR TITLE
improve logging sample

### PR DESCRIPTION
**Pull request by sclassen**

### CODE REVIEW

File: logging-sample/README.adoc [hunk 1]
```diff
@@ -3,59 +3,99 @@
...
+1. In the popup select the "global" checkbox and give the input a title (i.e. GELF UDP)
+1. Click "save".
++
 image::.readme/graylog-gelf-input.png[]
-
-Once Graylog is started the 2 Spring Boot application that are part of the sample can be started.
-Go to the `server-spring-log4j` or `server-spring-logback` subfolder and use maven to start the server:
-`mvn spring-boot:run`
+1. Click "Show received messages" to the right of the newly added input.
+1. Click on the little play symbol in the top right corner to enable auto refresh of the page.
```
Comment (hendrikebbers) [inline logging-sample/README.adoc:52]: I assume that should be:
```suggestion
2. Click on the little play symbol in the top right corner to enable auto refresh of the page.
```

File: logging-sample/README.adoc [hunk 2]
```diff
@@ -3,59 +3,99 @@
...
+INFO::
+If the ELK stack fails to start with the message `max virtual memory areas vm.max_map_count [65530] is too low, increase to at least [262144]` or similar than have a look at https://github.com/docker-library/elasticsearch/issues/111[elasticsearch issue #111]
+
+=== Configuring Kibana
+
+The ELK stack is ready to receive log messages without a configuration.
+In order to browse the messages a little configuration is needed:
+
+1. Launch the server-spring-log4j or the server-spring-logback example (`mvn spring-boot:run`).
+1. Visit http://localhost:5601[http://localhost:5601].
```
Comment (hendrikebbers) [inline logging-sample/README.adoc:82]: same as above: 1,2,3...

Comment (madmas) [reply logging-sample/README.adoc:82]: no, this automatically renders to a correctly numbered list, no need to count by hand ;-)

File: logging-sample/README.adoc (hunk 1 above)
Comment (madmas) [reply logging-sample/README.adoc:52]: no, this automatically renders to a correctly numbered list, no need to count by hand ;-)

File: logging-sample/README.adoc (hunk 2 above)
Comment (madmas) [reply logging-sample/README.adoc:82]: see https://github.com/rico-projects/rico-samples/blob/42174900b067fbf46ea6991d7622c26fa34787a5/logging-sample/README.adoc ;-)

**Pull request merged**